NewSpring/Rock
# Removed max height style definition for dropdown menus

## DESCRIPTION

This PR removes max height style definition for dropdown menus so that the select/cancel buttons are always visible. I've added the style to the site overrides so we could get the fix in immediately. Once this is merged we can remove it.

### How do I test this PR?

- https://newspring.cc/groups/leader/1563921
- Add a Group Member
- Select button should be visible at all times

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
